resources/js/Components/AuthWidget.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-    <div class="auth-widget" v-if="user">
+    <div class="auth-widget"> 
+        <div class="auth-widget" v-if="user">
```

Comment:
There's probably a nicer way to do this that also doesn't break the styles but at this point in time i'm not aware of it

Reply:
yeah it look weird.